### ISSUE TITLE
Node babel application. stripe.create.customer failing

### ISSUE THREAD

**Issue by mzahidriaz** [stripe/stripe-node#244]
I am developing application using node babel.
Strangely stripe is failing on create customer.

```
stripe.customers.create({

}, function(err, customer) {

}
```

failing with `error on creating object from null or undefined`

**Comment by remistr** [comment]
@mzahidriaz : Sounds like `stripe` or `stripe.customers` isn't defined properly here. I'd recommend double-checking to make sure that you installed the Stripe node library properly on your system first

**Comment by mzahidriaz** [comment]
If I log stripe or stripe.customer it prints correctly on console.

**Comment by matthewarkin** [comment]
@mzahidriaz what if you console.log err and customer?

**Comment by mzahidriaz** [comment]
After couple of hours hit & trials. I found the issue.

I created a separate node project and stripe was working fine there.

I am using [NPM CS](https://www.npmjs.com/package/cs) for my node project.

Before calling the API stripe generates the user-agent by issuing a command in child process to get machine name `uname a`.
That command was failing and producing the above mentioned error.

20 minutes of  tasks took 6 hours to figure out whats causing the issue.
